.github/main.workflow

```diff
@@ -11,7 +11,7 @@ action "Docker Registry" {
 action "build" {
   uses = "actions/docker/cli@master"
   needs = ["Docker Registry"]
-  args = "build -t docker.pkg.github.com/kubernetes-sigs/cluster-api-provider-docker:latest ."
+  args = "build -t docker.pkg.github.com/kubernetes-sigs/cluster-api-provider-docker/manager:latest ."
```

Comment:
```suggestion
  args = "build -t docker.pkg.github.com/kubernetes-sigs/cluster-api-provider-docker/capd-manager:latest ."
```

Reply:
that's already in the repository name, do you think it's unclear as is?

Reply:
I think it's clear and would leave the image name as `manager`

Reply:
ah i see your note about consistency in slack.

I do not see the consistency between CAPA/CAPI and I think this is a chance to create patterns that are less redundant than what already exist

Reply:
The image is called `capd-manager` in the `publish-manager.sh`
Proposing to be consistent w/ the name

Reply:
I think it makes sense in publish-manager to keep it as capd-manager because gcr.io repositories are structured as `gcr.io/<project-name>/<image-name>:<tag>`. Unfortunately the project-name is almost never going to be cluster-api-provider-docker related and thus the prefix capd- makes sense in that context.

Reply:
I am cool w/ that!